run-local/docker-compose.yaml

```diff
@@ -157,6 +159,12 @@ services:
     ports:
     - "16686:16686"
     - "4317:4317" #  if you want to send traces from host
+    - "5775:5775"
```

Comment:
those mappings are probably not needed - the goal is to use open telemetry port only in the examples